pkg/tsdb/errors/errors.go

```diff
@@ -0,0 +1,18 @@
+package errors
```

Comment:
I cannot see any code referring to creating a `BadRequest` so this file should probably be removed.